sources/modules/compilation-model/separate-compilation.md

```diff
@@ -0,0 +1,103 @@
+## Module name: Separate Compilation
+
+_Skeleton descriptions are typeset in italic text,_
+_so please don't remove these descriptions when editing the topic._
+
+### Overview
+
+_Provides a short natural language abstract of the module’s contents._
+_Specifies the different levels of teaching._
+
+This module outlines the issues involved in using separate compilation on multiple program files.
+
+------------------------------------------------------------------------
+Level             Objective
+----------------- ------------------------------------------------------
+Foundational:      Awareness of the existence of separate compilation
+
+Main:              Awareness of supporting mechanisms and tools
+
+Advanced:          Awareness of technicalities and tools
+
+------------------------------------------------------------------------
+
+### Motivation
+
+_Why is this important?_
+_Why do we want to learn/teach this topic?_
+
+Any non-trivial program will span more than one source file. This makes separate compilation necessary. Separate compilation also leads to quick build times in projects under development.
+
+### Topic introduction
+
+_Very brief introduction to the topic._
+
+### Foundational: Using
```

Comment:
```suggestion
### Foundational: Awareness of the existence of separate compilation
```

Reply:
Isn't the term "using" mandated by the template?

Reply:
That was resolved in today's meeting.

Reply:
Better caption